openstack_exporter/collectors/cinderbackend.py

```diff
@@ -231,6 +236,24 @@ def _parse_pool_data(self, pool, volume_type):
         data['driver_version'] = caps['driver_version']
         return data
 
+    def _debug_gauge(self, gauge, name, value, shard, backend, pool):
+        LOG.debug('({}/{}/{})-{} = {}'.format(
+            shard, backend, pool, name, value
+        ))
```

Comment:
Why are you not using f-strings here? Because that's just copy-and-paste? The new log entries you introduce in this commit use f-strings.

Reply:
done